server/planning/events/events_schema.py

```diff
@@ -195,7 +195,31 @@
         "mapping": {"properties": {"qcode": not_analyzed, "name": not_analyzed}},
     },
     # Content metadata
-    "subject": metadata_schema["subject"],
+    "subject": {
+        "type": "list",
+        "mapping": {
+            "type": "nested",
+            "include_in_parent": True,
+            "dynamic": False,
+            "properties": {
+                "qcode": string_with_analyzer,
+                "name": string_with_analyzer,
+                "scheme": string_with_analyzer,
+                "translations": {
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "object",
+                            "properties": {
+                                "fr": string_with_analyzer,
+                                "nl": string_with_analyzer,
```

Comment:
this would make it work only for belga, would be better probably to define dynamic mapping here to allow any language to be present